### PR TITLE
Add summary section to each case

### PR DESCRIPTION
**Pull request by XiaSun-Atmos**
**This PR does the following:**
1. Add GFSv15 and GFSv16 list of biases to the Intro
2.Add references wherever possible
3.Add summary section to each case
4.Add timeline of physics frozen for the different model versions

### CODE REVIEW

File: docs/source/2017DenverPBL.rst [hunk 1]
```diff
@@ -82,6 +82,11 @@ The Skew-T Log-P plot is created using the script adapted from `SHARPpy <https:/
...
       Skew-T Log-P plot from observed and simulated sounding profiles. Indices including K-index and lapse rate are shown in the bottom.
 
-    * GFS.v16.0.10 underestimates the temperature inversion strength at 800 hPa with a warmer near surface temperature and a colder temperature at the top of inversion.  
+    * GFS.v16.0.10 underestimates the temperature inversion strength at 800 hPa with a warmer near surface temperature and a colder temperature at the top of inversion. 
+ 
+......................
+Summary and Discussion
+......................
 
+The 2017 Denver Inversion results show that the GFS model lacks improvement in forecasting the boundary layer temperature inversion, with a warmer near-surface temperature and a colder temperature at the top of the inversion.
```
Comment (climbfuji) [inline docs/source/2017DenverPBL.rst]: either "lacks skill" or something like GFS v16 not showing improvements over GFS v15

File: docs/source/2018Michael.rst [hunk 2]
```diff
@@ -116,4 +116,12 @@ Hurricane Track and Intensity
     * The maximum surface wind speed at the vortex center in GFS.v16.0.10 reaches to 90 kts, which is 50 kts lower than Best Track data.
     * The minimum sea level pressure from GFS.v16.0.10 is 950 hPa, versus 919 hPa from Best Track data. 
 
+......................
+Summary and Discussion
+......................
 
+The physics compset of GFS.v16beta in MRW.v1.0 does not show improvements compared with the compset of GFS.v15p2 in simulating Hurricane Michael track and intensity. The physics for MRW_GFS.v16beta was frozen in Jan 2020 for the public release of UFS MRW App version 1.0. Several major changes have been made to the GFS model after that, which include the modified background diffusivity as a function of stability in the PBL scheme (`Yang 2020 <https://ufscommunity.org/wp-content/uploads/2020/10/UFS_Webnair_GFSv16_20201022_FanglinYang.pdf>`_). In GFS.v16.0.10 (GitHub tag created in Jun 2020), the simulated hurricane landfall location and hurricane intensity is slightly more close to Best Track data compared to MRW_GFSv15p2.
```
Comment (climbfuji) [inline docs/source/2018Michael.rst]: "slightly more close ..." --> "closer to the Best Track data, compared to MRW_GFS.v15p2 and MRW_GFS.v16beta"

Also: the list of changes could include (if not mentioned elsewhere) more vertical levels, several tuning parameters, ...

File: docs/source/2019Barry.rst [hunk 3]
```diff
@@ -147,6 +147,16 @@ Hovmöller diagram of 850 hPa WS after Landfall
...
-    * The low-level wind speed in UFS weather model is smaller than GFS_ANL.
+    * The low-level wind speed in GFS model is smaller than GFS_ANL.
     * The MRW_GFSv16beta has the largest inner core size. MRW_GFS15p2 has similar inner core sizes and GFS_ANL.
     * Hurricane intensity attenuates faster in the model compared with Best Track after landfall.
+      
+......................
+Summary and Discussion
+......................
+
+The exacerbated right-of-track bias exists in both GFS.v16.0.10 and MRW_GFSv16beta compared with GFSv15p2. The right-of-track bias is suspected to associate with overactive convection and vortex tilt, which induces convergence towards the excessive convection and leads to track bias (`Lybarger et al. 2020 <https://dtcenter.org/sites/default/files/events/2020/2-lybarger-nick.pdf>`_).
```
Comment (climbfuji) [inline docs/source/2019Barry.rst]: Please revisit this sentence. There is convection in the first and second half, i.e. overactive convection induces convergence towards excessive convection. Also, "is suspected to associate with" should be "is likely associated with"

File: docs/source/2019MemHeat.rst [hunk 4]
```diff
@@ -118,3 +118,11 @@ Surface Temperature
       2-m temperature (F)
 
     * GFS.v16.0.10 successfully captures the high temperatures across the Southeast.
+
+......................
+Summary and Discussion
+......................
+
+MRW_GFSv16beta generates cold bias for 2-m temperature over most of CONUS during the 2019 Memorial Day Heat Wave event. However, both MRW.GFSv15p2 and GFS.v16.0.10 simulate the extreme temperature well over the Southeastern U.S. This means that the cold bias of this extreme temperature case in the GFS model is already fixed in the following physics developments (see `Timeline of physics frozen`_ for the different model versions).
```
Comment (climbfuji) [inline docs/source/2019MemHeat.rst]: generates a cold bias

File: docs/source/2020EasterStorm.rst [hunk 5]
```diff
@@ -137,3 +137,8 @@ Surface Temperature and Wind Speed
 
     * GFS.v16.0.10 generates higher magnitudes of surface gust over the Southeastern U.S. which is related to the progressive synoptic pattern compared with GFS_ANL.
 
+......................
+Summary and Discussion
+......................
+
+The upper-level trough simulated in MRW_GFSv15p2 moves too fast northeastwards compared with GFS analysis, leading to the cold bias over the region east of the trough axis. The upper-level trough does not form over the western U.S in MRW_GFSv16beta. In GFS.v16.0.10, the synoptic pattern progressiveness still exists, although to a lesser extent.
```
Comment (climbfuji) [inline docs/source/2020EasterStorm.rst]: ... leading to a cold bias over ...

File: docs/source/Intro.rst [hunk 6]
```diff
@@ -7,9 +7,25 @@
...
.... To facilitate the development and improvement of the NOAA operational model, NOAA's Global Forecast System (GFS) is shared with the broad community through the `Unified Forecast System (UFS) <https://ufscommunity.org/>`_ applications. GFS version 15 (v15) was in operation in June 2019 and the planned date for GFS version 16 in operation is in Feb 2021 (`Yang 2020 <https://ufscommunity.org/wp-content/uploads/2020/10/UFS_Webnair_GFSv16_20201022_FanglinYang.pdf>`_). The UFS Medium-Range Weather (MRW) Application (App) provides the source code of GFS, including physics schemes for GFS.v15 (CompSet ``GFS.v15p2``) and additional developmental schemes for GFS.v16 (CompSet ``GFS.v16beta``). The most up-to-date source codes for GFS.v16 can be accessed from the `ufs-weather-model <https://github.com/ufs-community/ufs-weather-model>`_ GitHub repository. The known biases and development priorities related to GFS.v15 (`Stan et al. 2019 <https://drive.google.com/file/d/1rdFPbY28d7cRrcShy0uo4Mtqwh3BSzYg/view>`_) and GFS.v16 (recently presented by `Yang 2020 <https://ufscommunity.org/wp-content/uploads/2020/10/UFS_Webnair_GFSv16_20201022_FanglinYang.pdf>`_) have been identified and listed below:
```
Comment (climbfuji) [inline docs/source/Intro.rst]: I would move the sentence "GFS version 15 ..." before the sentence starting with "The known biases and development ...". And also: GFS version 15 went into operations in June 2019, the operational implementation fo GFS version 16 is currently planned for February 2021.

File: docs/source/Intro.rst [hunk 7]
```diff
@@ -7,9 +7,25 @@
...
+    - Precipitation dry bias for moderate rainfall
+    - Struggle to capture boundary layer inversions    
+  .. group-tab:: GFS.v16
+
+    - Increased right-of-track bias for tropical cyclones (TC) at longer lead times at North Atlantic
+    - Larger TC False Alarm Rate in the western North Atlantic
+    - Exacerbated low instability bias, which is largely related to dry soil moisture
+    - Poor representation of radiation inversions
+   
+This document provides case studies evaluation results for GFS using the `UFS Medium Range Weather (MRW) App <https://ufs-mrweather-app.readthedocs.io/en/latest/index.html>`_ with the targeted audience as the broad community. These cases are also run using the NOAA Environmental Model Center (EMC) `Global Workflow <https://github.com/NOAA-EMC/global-workflow/wiki>`_, which points to the most up-to-date GFS codes, and are expected to provide insights for the on-going model development. Brief descriptions for MRW App and Global Workflow are below.
```
Comment (climbfuji) [inline docs/source/Intro.rst]: with the target audience being a broad community.

File: docs/source/Intro.rst [hunk 8]
```diff
@@ -18,11 +34,23 @@ This document provides case studies evaluation results for the UFS Weather Model
...
+
+The goal of this ongoing effort is to provide the community, as well as the physics development team, with a model testing platform where they can use the resources to conduct model runs and evaluate the model performance for representative meteorological cases. These case studies will provide insights for future model developments in aim to improve NOAA numerical weather forecasts. It should be noted that this is an ongoing effort that is aligned with the model public release and model development. Namely these evaluation results only apply to specific model versions. Timeline of physics frozen date in different model versions, including both UFS MRW APP CompSets and ufs-weather-model GitHub tags, are shown below:
```
Comment (climbfuji) [inline docs/source/Intro.rst]: ... developments and aim at improving ...

**Review by climbfuji — CHANGES_REQUESTED** [review]

File: docs/source/2019Barry.rst (hunk 3 above)
Comment (XiaSun-Atmos) [reply docs/source/2019Barry.rst]: I split it into two sentences. I think they are referring to the same convection.

**Comment by XiaSun-Atmos** [comment]
Thank you @climbfuji  for the comments and suggestions. I have incorporated them into the recent commit.

**Review by climbfuji — APPROVED** [review]

**Pull request merged**